core/shared/src/main/scala/zio/metrics/MetricKey.scala

```diff
@@ -29,8 +29,8 @@ import zio.stacktracer.TracingImplicits.disableAutoTrace
 final case class MetricKey[+Type] private (
   name: String,
   keyType: Type,
-  tags: Set[MetricLabel] = Set.empty
```

Comment:
Shouldn't be changing the default parameter for tags here and elsewhere.

Reply:
I added `apply` method with default parameter for backward compatibility